examples/python/tools/trafilatura_example.py

```
@@ -0,0 +1,266 @@
+This example demonstrates how to create and use custom Trafilatura tools in PraisonAI
+for extracting high-quality content from web pages.
+
+This shows how to create custom tools that are not part of the core package.
+
+Install dependencies:
+pip install trafilatura newspaper3k requests beautifulsoup4
+"""
+
+import asyncio
+from praisonaiagents import Agent, Task, PraisonAIAgents
+from custom_trafilatura_tools import TrafilaturaTools, create_trafilatura_tools
```

Ruff (0.11.9)

`custom_trafilatura_tools.TrafilaturaTools` imported but unused

Remove unused import: `custom_trafilatura_tools.TrafilaturaTools`

(F401)